src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html

```diff
@@ -15,58 +15,43 @@
     <article class="download-content">
         <section class="fw-wrapper">
             <div class="row">
-                <div class="small-12 large- 6 columns">
-
+                <div class="small-12 columns">
                     <p>
                         You can use Maven or sbt to build services you create with the Lagom Java API.
-                        (If you aren’t using one of these tools already, see this information to 
-                        help you choose
-                  between <a href="http://www.scala-sbt.org/0.13/docs/index.html">sbt</a> and  
-                        <a href="https://maven.apache.org">Maven</a>.) The Lagom development environment runs on Windows, Mac, and Linux operating systems.
-                     Try it from the command line to get started and later integrate your 
-                     project into an IDE. Maven and sbt plugins make it especially easy to integrate the Lagom development environment with Eclipse or IntelliJ IDEA.</p> 
-                    
-                    <p>The following screen shows a successful run of the Lagom development 
-                      environment with Maven:
-                    </div>
-                  <br/>
-                                      
-                     <div class="small-12 large-6 columns">
-                    
-                           <img src="@context.path/images/ui/maven-java-run-all.png"/>
-                           <br/>
-                           <p></p>
-                           <p></p>
-                  
-                     </div>
-                     
-                     
-                    
-                     
-                     <div class="small-12 columns">
+                        (If you aren’t using one of these tools already, see this information to
+                        help you choose between <a href="http://www.scala-sbt.org/0.13/docs/index.html">sbt</a> and
+                        <a href="https://maven.apache.org">Maven</a>.) The Lagom development environment runs on
```

Comment:
Should the `.` be outside the parens in the sinppet

````
<a href="https://maven.apache.org">Maven</a>.) 
``` 
??

PS: I read the comments wrt UK/US/AUS variants and can't remember if this was discussed there. Feel free to ignore.